interfaces/IContentDecryption.h

```diff
@@ -35,6 +35,128 @@ namespace Exchange {
         virtual RPC::IStringIterator* Designators(const string& keySystem) const = 0;
         virtual RPC::IStringIterator* Sessions(const string& keySystem) const = 0;
     };
+
+
+
+    class DataExchange : public WPEFramework::Core::SharedBuffer {
```

Comment:
Guess this was in the old file already but we try to use to minimize the unnecessary inclusion of the WPEFramework namespace. At some point we might need to rename this product so we try to minimize the number of cases we need to replace :) So could you please remove it if possible?